xonsh/platform.py

```diff
@@ -240,7 +246,12 @@ def scandir(path):
                      'usr\\share\\bash-completion\\completions'),
         os.path.join(GIT_FOR_WINDOWS_PATH,
                      'mingw64\\share\\git\\completion\\git-completion.bash'))
-
+    PATH_DEFAULT = tuple(winreg.QueryValueEx(winreg.OpenKey(
+            winreg.HKEY_LOCAL_MACHINE,
+            r'SYSTEM\CurrentControlSet\Control\Session Manager\Environment'),
+        'Path')[0].split(os.pathsep))
```

Comment:
OK. There is a small bug in `locate_binary()` which prevent this.  I have made a PR for it. #1215

But that aside I don't think Anaconda Python can start on windows unless $PATH exists. Probably Continuum added something to `lib/site.py`. The folder `Library\bin` is prepended to the path on startup. However, it doesn't seem to be an issue with the original python source.